src/pages/staticPages/tools/oncoprinter/OncoprinterImportUtils.ts

```diff
@@ -142,7 +143,8 @@ function getOncoprinterGeneticInputLine(parsed: OncoprinterGeneticInputLine) {
                 line.push('PROT');
                 break;
             case AlterationTypeConstants.STRUCTURAL_VARIANT:
-                line.push('STRUCTURAL VARIANT');
+                line.push(parsed.eventInfo || 'structural_variant');
```

Comment:
maybe comment why it would be missing eventInfo?

Reply:
I just wanted to do this as a catch in case there's missing data. would eventInfo always exist for structural variants?